ExportScripts/export_standalone_monolithic_windows.py

```diff
@@ -0,0 +1,285 @@
+
+
+# EXPORT SCRIPT STARTS HERE!
+
+
+if __name__ == "__main__":
+    
+
+    parser = argparse.ArgumentParser(prog='Exporter for MultiplayerSample on windows',
+                                 description = "Exports O3DE's MultiplayerSample to the desired install directory in project layout...")
+    parser.add_argument('-ps', '--project-path', type=pathlib.Path, required=True, help='Path to the intended O3DE project.')
+    parser.add_argument('-egn', '--engine-path', type=pathlib.Path, required=True, help='Path to the intended O3DE engine copy.')
+    parser.add_argument('-out', '--output-path', type=pathlib.Path, required=True, help='Path that describes the final resulting Release Directory path location.')
+    parser.add_argument('-cfg', '--config', type=str, default='profile', choices=['release', 'profile'], help='The CMake build configuration to use when building project binaries.')
+    parser.add_argument('-ll', '--log-level', default='INFO',
+                        choices=['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'],
+                        help="Set the log level")
+    parser.add_argument('-bnmt', '--build-non-mono-tools', action='store_true')
+    parser.add_argument('-nmbp', '--non-mono-build-path', type=pathlib.Path, default=None)
+    parser.add_argument('-mbp', '--mono-build-path', type=pathlib.Path, default=None)
+    parser.add_argument('-zip', '--zip-output', action='store_true', help='This option places the final output of the build into a zipped archive')
+    args = parser.parse_args()
+
+    logging.getLogger().setLevel(args.log_level)
+
+    non_mono_build_path = (args.engine_path) / 'build' / 'non_mono' if args.non_mono_build_path is None else args.non_mono_build_path
+    mono_build_path = (args.engine_path) / 'build' / 'mono' if args.mono_build_path is None else args.mono_build_path
+
+    #validation
+    assert args.engine_path.is_dir() and (args.engine_path / 'engine.json').is_file(), "Invalid Engine path provided!"
+    assert args.project_path.is_dir() and (args.project_path / 'project.json').is_file(), "Invalid Project path provided!"
+
+    #commands are based on 
+    #https://github.com/o3de/o3de-multiplayersample/blob/development/Documentation/PackedAssetBuilds.md
+
+    
+    #Build o3de-multiplayersample and the engine (non-monolithic)
+    if args.build_non_mono_tools:
+        process_command(['cmake', '-S', '.', '-B', str(non_mono_build_path), '-DLY_MONOLITHIC_GAME=0', f'-DLY_PROJECTS={args.project_path}'], cwd=args.engine_path)
+
+        process_command(['cmake', '--build', str(non_mono_build_path), '--target', 'Editor', 'AssetBundler', 'AssetBundlerBatch', 'AssetProcessor', 'AssetProcessorBatch', '--config','profile'], cwd=args.engine_path)
+
+        asset_processor_batch_path = non_mono_build_path / 'bin' / 'profile' / 'AssetProcessorBatch'
+        process_command([asset_processor_batch_path, '--project-path', args.project_path], cwd=args.engine_path)
```

Comment:
Another option is to build MultiplayerSample.Assets target with cmake which should build the assets for you.